code/modules/clothing/head/helmet.dm

```diff
@@ -257,6 +257,58 @@
 		usr.update_action_buttons()
 		return 1
 
+/obj/item/clothing/head/helmet/hazcap
+	name = "CBRN Helmet"
```

Comment:
CBRN helmet - lowercase so it's same as the other helmets

Reply:
fixed